### PR TITLE
add 4.09.05 changelog

### PR DESCRIPTION
**Pull request by Crabcyborg**
Adding in the change log for 4.09.05.

Leaving out the copies update as it still hasn't merged, and I feel like this update is important (and the copies update isn't too necessary).

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,4 +1,12 @@
 == Changelog ==
+= 4.09.05 =
+* Fix: Autofill was flagging form submissions as spam with Honeypot in some browsers.
+* Fix: Repeater fields were occasionally filling in with the wrong value.
```
Comment (stephywells) [inline changelog.txt:4]: @Crabcyborg I've been doing two separate changelogs. So only the lite features would be included here, and the pro features would be in the changelog on our site. 

So datepickers, repeaters, file uploads, calculations are all pro features.

For the security note, let's add a bit more info without being too specific.

Comment (Crabcyborg) [reply changelog.txt]: Got it @stephywells what do you think about my new revision?

Comment (stephywells) [reply changelog.txt]: Looks great! But Honeypot is in lite, so let's keep that one in there.

Comment (Crabcyborg) [reply changelog.txt]: @stephywells I hadn't considered that! The fix was done in pro, and actually doesn't fix this issue for lite!

Comment (stephywells) [reply changelog.txt]: Nice catch!

**Pull request merged**